### PR TITLE
[zh] sync changes in docs/reference/setup-tools/kubeadm/ directory

### PR DESCRIPTION
**Pull request by howieyuen**
ref: #25247
sync files below, 11 in total
```
17	0	content/en/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md
0	2	content/en/docs/reference/setup-tools/kubeadm/kubeadm-version.md
6	4	content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md
0	2	content/en/docs/reference/setup-tools/kubeadm/kubeadm-upgrade.md
11	0	content/en/docs/reference/setup-tools/kubeadm/kubeadm-init-phase.md
0	4	content/en/docs/reference/setup-tools/kubeadm/kubeadm-token.md
1	0	content/en/docs/reference/setup-tools/kubeadm/kubeadm-upgrade-phase.md
41	35	content/en/docs/reference/setup-tools/kubeadm/kubeadm-join.md
11	5	content/en/docs/reference/setup-tools/kubeadm/kubeadm-config.md
0	3	content/en/docs/reference/setup-tools/kubeadm/kubeadm-reset.md
```

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md [hunk 1]
```diff
@@ -23,13 +23,23 @@ weight: 90
...
+
+
 ## kubeadm alpha certs renew {#cmd-certs-renew}
 
 <!--
 You can renew all Kubernetes certificates using the `all` subcommand or renew them selectively.
 For more details about certificate expiration and renewal see the [certificate management documentation](/docs/tasks/administer-cluster/kubeadm/kubeadm-certs/).
 -->
-使用 `all` 子命令来更新所有 Kubernetes 证书或有选择性地更新它们。有关证书到期和续订的更多详细信息，请参见[证书管理文档](/docs/tasks/administer-cluster/kubeadm/kubeadm-certs/)。
+使用 `all` 子命令来更新所有 Kubernetes 证书或有选择性地更新它们。有关证书到期和续订的更多详细信息，请参见[证书管理文档](/zh/docs/tasks/administer-cluster/kubeadm/kubeadm-certs/)。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md:42]: 最好适当换行。

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md [hunk 2]
```diff
@@ -59,6 +69,18 @@ to enable the automatic copy of certificates when joining additional control-pla
 {{< tab name="certificate-key" include="generated/kubeadm_alpha_certs_certificate-key.md" />}}
 {{< /tabs >}}
 
+## kubeadm alpha certs generate-csr {#cmd-certs-generate-csr}
+
+<!-- 
+This command can be used to generate certificate signing requests (CSRs) which
+can be submitted to a certificate authority (CA) for signing.
+-->
+该命令可用于生成证书签名请求（CSR），可以将其提交给证书颁发机构（CA）进行签名。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md:78]:
```suggestion
该命令可用于生成证书签名请求（CSR），CSR 可以被提交给证书颁发机构（CA）进行签名。
```

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-init.md [hunk 3]
```diff
@@ -71,64 +68,87 @@ following steps:
...
-一旦控制平面的 Pod 都运行起来， `kubeadm init` 的工作流程就继续往下执行。
+   一旦控制平面的 Pod 都运行起来， `kubeadm init` 的工作流程就继续往下执行。
 
 <!--
 1. Apply labels and taints to the control-plane node so that no additional workloads will
-run there.
+   run there.
 -->
-1. 对控制平面节点应用 labels 和 taints 标记以便不会在它上面运行其它的工作负载。
+5. 对控制平面节点应用 labels 和 taints 标记以便不会在它上面运行其它的工作负载。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/kubeadm-init.md:92]:
```suggestion
5. 对控制平面节点应用标签和污点标记以便不会在它上面运行其它的工作负载。
```

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-init.md [hunk 4]
```diff
@@ -71,64 +68,87 @@ following steps:
...
    - 创建一份 ConfigMap 提供添加集群节点所需的信息，并为该 ConfigMap 设置相关的 RBAC 访问规则。
+
    - 使得 Bootstrap Tokens 可以访问 CSR 签名 API。
-   - 对新的 CSR 请求配置为自动签发。
 
-<!--
-See [kubeadm join](/docs/reference/setup-tools/kubeadm/kubeadm-join/) for additional info.
--->
-查阅[kubeadm join](/zh/docs/reference/setup-tools/kubeadm/kubeadm-join/)文档以获取更多信息。
+   - 对新的 CSR 请求配置为自动签发。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/kubeadm-init.md:127]:
```suggestion
   - 配置自动签发新的 CSR 请求
```

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-init.md [hunk 5]
```diff
@@ -209,28 +229,33 @@ The config file is still considered beta and may change in future versions.
...
 可以使用 [kubeadm config print](/zh/docs/reference/setup-tools/kubeadm/kubeadm-config/)命令打印出默认配置。
 
+如果你的配置没有使用最新版本，
 **推荐**使用 [kubeadm config migrate](/zh/docs/reference/setup-tools/kubeadm/kubeadm-config/)
-命令将旧的 `v1beta1` 版本的配置迁移到 `v1beta2` 版本。
+命令进行迁移。
 
-获取 `v1beta2` 版本配置中每个字段的细节说明，查看我们的
-[API 参考页面](https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta2)。
+有关配置的字段和用法的更多信息，你可以导航到我们的 API 参考页面并从[列表](https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm#pkg-subdirectories)中选择一个版本。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/kubeadm-init.md:258]: 适当换行

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join.md [hunk 6]
```diff
@@ -39,85 +39,159 @@ This action consists of the following steps:
...
+1. Downloading certificates shared among control-plane nodes from the cluster
+  (if explicitly requested by the user).
+
+1. Generating control-plane component manifests, certificates and kubeconfig.
+
+1. Adding new local etcd member.
+
+1. Adding this node to the ClusterStatus of the kubeadm cluster.
+-->
+对于控制平面节点，执行其他步骤：
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join.md:73]:
```suggestion
对于控制平面节点，执行额外的步骤：
```

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join.md [hunk 7]
```diff
@@ -39,85 +39,159 @@ This action consists of the following steps:
...
+1. 生成控制平面组件清单、证书和 kubeconfig。
+
+1. 添加新的本地 etcd 成员。
+
+1. 将此节点添加到 kubeadm 集群的 ClusterStatus。
+
+<!-- 
+### Using join phases with kubeadm {#join-phases}
+-->
+### 通过 kubeadm 使用 join 阶段 {#join-phases}
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join.md:86]:
```suggestion
### 使用 kubeadm 的 join phase 命令 {#join-phases}
```

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join.md [hunk 8]
```diff
@@ -39,85 +39,159 @@ This action consists of the following steps:
...
+kubeadm join phase kubelet-start --help
+```
+
+<!-- 
+Similar to the [kubeadm init phase](/docs/reference/setup-tools/kubeadm/kubeadm-init/#init-phases)
+command, `kubeadm join phase` allows you to skip a list of phases using the `--skip-phases` flag.
+
+For example:
+-->
+类似于 [kubeadm init phase](/zh/docs/reference/setup-tools/kubeadm/kubeadm-init/#init-phases)命令，`kubeadm join phase` 允许你使用 `--skip-phases` 标志跳过阶段列表。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join.md:117]: 适当换行

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join.md [hunk 9]
```diff
@@ -39,85 +39,159 @@ This action consists of the following steps:
...
 The CA key hash has the format `sha256:<hex_encoded_hash>`. By default, the hash value is returned in the `kubeadm join` command printed at the end of `kubeadm init` or in the output of `kubeadm token create --print-join-command`. It is in a standard format (see [RFC7469](https://tools.ietf.org/html/rfc7469#section-2.4)) and can also be calculated by 3rd party tools or provisioning systems. For example, using the OpenSSL CLI:
 -->
-
 CA 键哈希格式为 `sha256:<hex_encoded_hash>`。
-默认情况下，在 `kubeadm init` 最后打印的 `kubeadm join` 命令或者 `kubeadm token create --print-join-command` 的输出信息中返回哈希值。
-它使用标准格式 (请参考 [RFC7469](https://tools.ietf.org/html/rfc7469#section-2.4)) 并且也能通过第三方工具或者驱动系统进行计算。
+默认情况下，在 `kubeadm init` 最后打印的 `kubeadm join` 命令
+或者 `kubeadm token create --print-join-command` 的输出信息中返回哈希值。
+它使用标准格式 (请参考 [RFC7469](https://tools.ietf.org/html/rfc7469#section-2.4)) 
+并且也能通过第三方工具或者驱动系统进行计算。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join.md:161]:
```suggestion
并且也能通过第三方工具或者制备系统进行计算。
```

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join.md [hunk 10]
```diff
@@ -144,11 +218,14 @@ kubeadm join --discovery-token abcdef.1234567890abcdef --discovery-token-ca-cert
 
 **劣势：**
  - CA 哈希通常在主节点被提供之前是不知道的，这使得构建使用 kubeadm 的自动化配置工具更加困难。
-   通过预先生成CA，您可以解决这个限制。
+   通过预先生成CA，你可以解决这个限制。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join.md:221]:
```suggestion
   通过预先生成CA，你可以解除这个限制。
```

**Review by tengqm — COMMENTED** [review]
NICE JOB.
非常细心严谨。
谢谢！

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join.md (hunk 10 above)
Comment (howieyuen) [reply content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join.md:221]: done

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join.md (hunk 9 above)
Comment (howieyuen) [reply content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join.md]: resloved

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join.md (hunk 8 above)
Comment (howieyuen) [reply content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join.md]: resolved

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join.md (hunk 7 above)
Comment (howieyuen) [reply content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join.md]: nice translation 👍

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join.md (hunk 6 above)
Comment (howieyuen) [reply content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join.md]: resovled

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-init.md (hunk 5 above)
Comment (howieyuen) [reply content/zh/docs/reference/setup-tools/kubeadm/kubeadm-init.md]: done

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-init.md (hunk 4 above)
Comment (howieyuen) [reply content/zh/docs/reference/setup-tools/kubeadm/kubeadm-init.md]: done

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-init.md (hunk 3 above)
Comment (howieyuen) [reply content/zh/docs/reference/setup-tools/kubeadm/kubeadm-init.md]: resolved

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md (hunk 2 above)
Comment (howieyuen) [reply content/zh/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md]: resolved

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md (hunk 1 above)
Comment (howieyuen) [reply content/zh/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md]: done

**Comment by tengqm** [comment]
THANKS
/lgtm
/approve

**Pull request merged**